config/owasp/suppressions.xml

```diff
@@ -35,4 +35,11 @@
         <packageUrl regex="true">^pkg:maven/commons-fileupload/commons-fileupload@.*$</packageUrl>
         <cve>CVE-2023-24998</cve>
     </suppress>
+    <suppress>
```

Comment:
Could you please remove the outdated suppression block for snakeyaml beginning on line 10